addons/website_forum/static/src/js/website_forum.js

```diff
@@ -404,59 +424,88 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
      * @private
      * @param {Event} ev
      */
-    _onValidationQueueClick: function (ev) {
+    _onValidationQueueClick: async function (ev) {
         ev.preventDefault();
-        var $link = $(ev.currentTarget);
-        $link.parents('.post_to_validate').hide();
-        $.get($link.attr('href')).then(() => {
-            var left = $('.o_js_validation_queue:visible').length;
-            var type = $('h2.o_page_header a.active').data('type');
-            $('#count_post').text(left);
-            $('#moderation_tools a[href*="/' + type + '_"]').find('strong').text(left);
-            if (!left) {
-                this.$('.o_caught_up_alert').removeClass('d-none');
-            }
-        }, function () {
-            $link.parents('.o_js_validation_queue > div').addClass('bg-danger text-white').css('background-color', '#FAA');
-            $link.parents('.post_to_validate').show();
-        });
+        const linkClicked = ev.currentTarget;
+        const parentPost = this._findParent(linkClicked, '.post_to_validate');
+        if (!parentPost) {
+            return;
+        }
+        parentPost.classList.add('d-none');
+        let ok;
+        try {
```

Comment:
Not sure what this code is doing (this whole block), but anyway it deserves a comment somewhere...